### PR TITLE
[Post Update] @property Multiple Declarations

### LINKED ISSUE #4260: content: css @property
### What page(s) need to be updated?

* https://web.dev/at-property/
* https://github.com/GoogleChrome/web.dev/blob/master/src/site/content/en/blog/at-property/index.md

### Why is this update needed?

Missing syntax for multiple properties / list of properties.

Is it:

```css
@property --xxx,
@property --yyy {
}
```

or:

```css
@property --xxx, --yyy {
}
```

or something else?

@una

### PR DESCRIPTION
**Pull request by una**
Fixes #4260

Changes proposed in this pull request:

- Adds syntax and explainer for multiple property declarations in the same line for `@property` blog post

### CODE REVIEW

File: src/site/content/en/blog/at-property/index.md [hunk 1]
```diff
@@ -203,6 +203,31 @@ This will now enable that smooth gradient transition.
 </figure>
 -->
 
+## Multiple Declarations
```
Comment (kaycebasques) [inline src/site/content/en/blog/at-property/index.md:206]:
```suggestion
## Multiple declarations
```

File: src/site/content/en/blog/at-property/index.md [hunk 2]
```diff
@@ -203,6 +203,31 @@ This will now enable that smooth gradient transition.
...
+
+Another neat feature of `@property` is that you can declare multiple new custom properties at the same time. For example, if you wanted to set multiple values with the same syntax but a unique `initial-value`, you could write something like:
```
Comment (kaycebasques) [inline src/site/content/en/blog/at-property/index.md:208]:
```suggestion
Another neat feature of `@property` is that you can declare multiple
new custom properties at the same time. For example, if you wanted
to set multiple values with the same syntax but a unique `initial-value`,
you could write something like:
```

**Review by kaycebasques — APPROVED** [review]

**Comment by una** [comment]
Reverting this commit. The multi-declaration syntax is not valid. [Demo](https://codepen.io/una/pen/24f72fc035d47874900024d2c517a462)

@ramiy, I had checked your codebase to confirm, but upon second glance, it looks like `postcss-combine-duplicated-selectors` is doing the combination. I should have been more careful. Please update [MDN](https://github.com/mdn/content/pull/4632) as well.

CC @rachelandrew

**Comment by LeaVerou** [comment]
> Reverting this commit. The multi-declaration syntax is not valid. [Demo](https://codepen.io/una/pen/24f72fc035d47874900024d2c517a462)
> 
> @ramiy, I had checked your codebase to confirm, but upon second glance, it looks like `postcss-combine-duplicated-selectors` is doing the combination. I should have been more careful. Please update [MDN](https://github.com/mdn/content/pull/4632) as well.
> 
> CC @rachelandrew

FWIW, I just opened an issue to actually allow this in CSS: https://github.com/w3c/css-houdini-drafts/issues/1058

**Pull request merged**